Setup/UpgradeSchema.php

```diff
@@ -46,6 +46,17 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
             ]
         );
 
+        $setup->getConnection()->addColumn(
```

Comment:
is reservedOrderId field indexed? if so maybe we should index this too

Reply:
Index is not required we just do set and get on a loaded model, no searches or queries of any kind.